src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_UsingStatement.cs

```diff
@@ -493,9 +494,11 @@ private BoundExpression MakeCallWithNoExplicitArgument(MethodArgumentInfo method
                 Debug.Assert(!assertParametersAreOptional || method.Parameters.All(p => p.IsOptional || p.IsParams));
                 Debug.Assert(method.ParameterRefKinds.IsDefaultOrEmpty);
             }
+
+            Debug.Assert(methodArgumentInfo.Arguments.All(arg => arg is not BoundConversion { ConversionKind: ConversionKind.InterpolatedStringHandler }));
```

Comment:
>Debug.Assert(methodArgumentInfo.Arguments.All(arg => arg is not BoundConversion { ConversionKind: ConversionKind.InterpolatedStringHandler })); [](http://example.com/codeflow?start=12&length=143)

Given the name of the containing method, I think the assert should be different. I.e. we should assert there are no arguments.

Reply:
Not quite. There can be an extension method argument, for example, and potentially arguments that are allowed to be omitted. The previous asserts cover these cases, and ensure that we only pass non-explicit parameters. Since there was never an explicit parameter, there should not be any interpolated conversions, so we assert that here.